### DISCUSSION TITLE
Add Cloudlog to heimdall

### DISCUSSION THREAD

**Discussion by resnikov** [magicbug/Cloudlog#1357] [Ideas / Feature Requests]
I use [heimdall ](https://heimdall.site/ )as a dashboard and think it would be good to get Cloudlog added to it. 

Happy to submit it but did not want to do it without asking first as it needs the logo.

[https://github.com/linuxserver/Heimdall-Apps](https://github.com/linuxserver/Heimdall-Apps)

**Comment by magicbug** [comment]
We dont have much of a real icon you'll find one at https://raw.githubusercontent.com/magicbug/Cloudlog/master/favicon.ico but feel free to see if they will add it.
